Makefile

```diff
@@ -8,13 +8,16 @@ ECHO1 = $(V:1=@ :)
 ECHO = $(ECHO1:0=@ echo)
 FUZZ_OUTPUT_DIR = $(shell pwd)/fuzz/output
 
-SOEXT := $(shell ruby -e 'puts RbConfig::CONFIG["SOEXT"]')
+SOEXT ?= $(shell ruby -e 'puts RbConfig::CONFIG["SOEXT"]')
 
 CPPFLAGS := -Iinclude $(CPPFLAGS)
 CFLAGS := -g -O2 -std=c99 -Wall -Werror -Wextra -Wpedantic -Wundef -Wconversion -Wno-missing-braces -fPIC -fvisibility=hidden $(CFLAGS)
```

Comment:
Both my Ubuntu Ruby and Windows RubyInstaller includes `-O3` in the `CFLAGS`, while this `Makefile` specifies `-O2`.
Could this matter?

Reply:
I don't think it's going to end up mattering truthfully — O2 keeps it from getting very long compile times.